components/BidderProfile.tsx

```diff
@@ -0,0 +1,553 @@
+      []),
+  ].join(' • ');
+
+  return (
+    <div style={{ backgroundColor: 'var(--dark-bg)' }}>
+      <div className="content">
+        <Stack direction="column" gap={-1}>
+          <Text variant="large-title" bold color="bright-text">
+            {parseAddress(address)}
+            <a
+              title={parseAddress(address)}
+              target="_blank"
+              rel="noreferrer"
+              href={`https://etherscan.io/address/${address}`}
+              className="external-link"
+            >
+              <ExternalLinkIcon />
+            </a>
+          </Text>
+          <div
+            style={{
+              height: '1rem',
+              maxWidth: '42ch',
+              overflow: 'hidden',
+              textOverflow: 'clip',
+              whiteSpace: 'nowrap',
+            }}
+          >
+            <Text variant="headline" bold color="low-text">
+              {domainNicknames.length < 42 ? (
+                domainNicknames
+              ) : (
+                <LiveLine text={domainNicknames + ' • '} />
+              )}
+            </Text>
+          </div>
+          <div
+            style={{
+              height: '1rem',
+            }}
+          >
+            {dapps && dapps.length > 0 && (
+              <Text variant="headline" bold color="low-text">
+                {dapps.map((dapp, index) => (
+                  <>
+                    <a
+                      href={`https://warpcast.com/${dapp.nickname}`}
+                      key={index}
+                      target="_blank"
+                      rel="noreferrer"
+                      className="social-link"
+                    >
+                      {dapp.type}
+                    </a>
+                    {index < dapps.length - 1 ? ' • ' : ''}
+                  </>
+                ))}
+              </Text>
+            )}
+          </div>
+          <div className="dotted-hr" />
+        </Stack>
+        <div className="balance">
+          <Stack direction="column" gap={1}>
+            <Text variant="title-1" bold color="low-text">
+              Balance
+            </Text>
+            <Stack direction="column" gap={-1}>
+              <Text variant="large-title" bold color="bright-text">
+                Ξ{formatEther(BigInt(balance?.eth || 0)).slice(0, 6)}
+              </Text>
+              <Text variant="headline" bold color="low-text">
+                {formatUsdPrice(Number(balance?.usd || '0'))}
+              </Text>
+            </Stack>
+          </Stack>
+          <div className="extra-space" />
+          <div className="nouns">
+            {nouns &&
+              nounSVGs &&
+              nounSVGs.length > 0 &&
+              nounSVGs.map(
+                (nounSVG, index) =>
+                  nounSVG && (
+                    <Link
+                      href={
+                        nouns[index].id % 10 === 0
```

Comment:
add a comment about why, also this can change in the future. So instead better always link to `/noun` and on that page redirect to opensea if auction not found